frontend/src/host-orchestrator/cmd/cvdserver-bootstrapper/go.mod

```diff
@@ -0,0 +1,3 @@
+module cvdserver-bootstrapper
```

Comment:
Please move `cvdserver-bootstrapper` to the `frontend/src` directory.

It shouldn't be under host-orchestrator since it's a different binary. The `cmd` directory should only exist if we create directories for the other modules, such as 'libs' or 'services', we only have 4 go modules IIRC so don't bother with that just yet.

Reply:
But I don't think the cvdserver-bootstrapper is a companion command to the host-orchestrator. The HO may depend on it, but won't execute it directly.

More importantly, I got a different interpretation of the purpose of the `/cmd` directory from the [golang-standards docs](https://github.com/golang-standards/project-layout). From reading that our directory structure should be
```
src
  |  cmd
  |    |  host-orchestrator
  |    |  host-operator
  |    |  cvdserver-bootstrapper
  |  pkg
  |    |  liboperator
  |  web
  |    |  webui
```
That would be a much more involved refactor that doesn't belong in this PR though.

Reply:
yes, that would be project structure for our Golang project if following those practices. So as you mention, it would be better to have it on other PR and for now follow the current structure. so I moved up `cvdserver-bootstrapper` to `frontend/src`.